src/api-routes/persons/{osuId}/jobs/{jobId}.js

```diff
@@ -48,6 +49,10 @@ const patch = async (req, res) => {
       return errorBuilder(res, 409, 'Job Id in path does not match job Id in body');
     }
 
+    if (changeReasonCode === 'AAHIR' && !studentEmployeeInd) {
+      return errorBuilder(res, 400, ['AAHIR change reason code cannot be used to update graduate job records']);
+    }
+
```

Comment:
Correct me if I am wrong, but I guess `AAHIR` can only be used to create graduate record, so a 400 should be returned in PATCH endpoint here if we see `AAHIR` no matter it is a graduate or a student.